CHANGES.rst

```diff
@@ -29,6 +29,10 @@ vizier
 - Change the type of raised error when the catalog is not found in ``Vizier.get_catalog_metadata``
   from ``IndexError`` to ``EmptyResponseError`` [#2980]
 
+sdss
+^^^^
+
+- Support new SDSS-V DR18 access URLs. [#3017]
```

Comment:
Let me think about that. If there are additional *non*-eFEDs spectra in DR18, then it is worth changing the default. Otherwise, I think it would be worth waiting and then bump to the default to DR19 when that is available.

Reply:
OK, let's make this independent of this PR then.

Reply:
Just to confirm: DR18 does not include any new eBOSS (*i.e.* non-eFEDS) spectra. In fact the eBOSS spectra appear to be identical between DR17 and DR18.